style.css

```diff
@@ -0,0 +1,177 @@
+:root {
+  --header-height: 100px;
+}
+
+html {
+  font-family: 'Merriweather', serif;
+  background: white;
+  color:black;
+  margin: 0px;
+  padding: 0px;
+  width: 100%;
+}
+
+body {
+  margin: 0px;
+  margin-top: var(--header-height);
+  padding: 0px;
+  width: 100%;
+}
+
+/* HEADER */
+
+header {
+  background-color: black;
+  color: white;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  top: 0;
+  position: fixed;
+  z-index: 4;
+  padding: 10px 20px 10px 20px;
```

Comment:
If you want to have the same padding for top/bottom and the same for left/right, you only have to specify two arguments, like this:
`padding: 10px 20px;`
then the first one applies for top/bottom and the second one for left/right 🥳